api/app/PasswordReset.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class PasswordReset extends Model
+{
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array
+     */
+    protected $fillable = [
+        'user_id', 'token', 'created_at',
+    ];
+
+    /**
+     * Indicates if the model should be timestamped.
+     *
+     * @var bool
+     */
+    public $timestamps = false;
```

Comment:
No timestamps, so you do ```Carbon::parse($datetime)```

Laravel has built something for that: https://laravel.com/docs/5.4/eloquent-mutators#date-mutators